server/src/dtos/asset-response.dto.ts

```diff
@@ -116,6 +122,7 @@ export function mapAsset(entity: AssetEntity, options: AssetMapOptions = {}): As
     livePhotoVideoId: entity.livePhotoVideoId,
     tags: entity.tags?.map(mapTag),
     people: peopleWithFaces(entity.faces),
+    faces: entity.faces?.map((a) => mapFacesWithoutPerson(a)),
```

Comment:
This duplicates the info in `people`. Could you filter to only include unassigned faces and maybe change `faces` to `facesWithoutPerson`?

Reply:
I think if `people` includes faces + people and `unassignedFaces` includes the rest of the faces, that would be ok. This is a bit annoying though, as this is adding more and more data to the asset response dto, which is used everywhere, including in the main timeline. We should probably change it to not return people/faces by default and only load those relations in the asset detail endpoint. Or, load those in a separate query like /people?assetId=<id> and /faces?assetId=<id>&state=unassigned

Reply:
That was what I was doing in the first place, using the `/face?id=`. For now, I filtered the faces to only include unassigned ones.

Reply:
> We should probably change it to not return people/faces by default and only load those relations in the asset detail endpoint.

The time bucket query doesn't have these relations, so I believe this is the current behavior.

Reply:
Oh, it looks like you are right. That works for me then.